airbyte-integrations/bases/source-acceptance-tests/src/main/resources/schemas/source_acceptance_test_inputs.yaml

```diff
@@ -0,0 +1,75 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/bases/source-acceptance-tests/src/main/resources/schemas/source_acceptance_test_inputs.yaml
+title: SourceAcceptanceTestConfig
+type: object
+description: SourceAcceptanceTestConfig
+required:
+  - connector_image
+  - tests
+properties:
+  connector_image:
+    type: string
+  tests:
+    type: array
+    items:
+      $ref: "#/definitions/TestConfiguration"
+    minItems: 1
+
+definitions:
+  TestConfiguration:
+    type: object
+    required:
+      - type
+    properties:
+      type:
+        type: string
+        enum:
+          - CORE
+          - FULL_REFRESH
+          - INCREMENTAL
+      core:
+        $ref: "#/definitions/CoreConfig"
+      fullRefresh:
```

Comment:
IIUC, all connectors should support `core` and `full_refresh` tests - should the `core` and `fullRefresh` configs be required?

Reply:
snake case to match everything else?

Reply:
> core is required, full refresh is not. Is it possible to require an item be present in an array? It wouldn't make sense to require core in this struct since this is a union type

you could just have core passed a separate field (i.e. not in the tests array), and then make that new field required. it's a little weird but it enforces what you want.

can full refresh and incremental tests rely on information passed in the core config? if so it makes that a bit clearer too.